### ISSUE TITLE
local-exec sleep provisioner fails on Windows

### ISSUE THREAD

**Issue by memes** [F5Networks/terraform-gcp-bigip-module#8]
When using the module on a Windows laptop, the BIG-IP module fails with an error because local-exec fails to execute sleep command.

```text
module.bigip[0].google_compute_instance.f5vm01: Provisioning with 'local-exec'...
module.bigip[0].google_compute_instance.f5vm01 (local-exec): Executing: ["cmd" "/C" "sleep 250"]
module.bigip[0].google_compute_instance.f5vm01 (local-exec): 'sleep' is not recognized as an internal or external command,
module.bigip[0].google_compute_instance.f5vm01 (local-exec): operable program or batch file.
╷
│ Error: local-exec provisioner error
│
│   with module.bigip[0].google_compute_instance.f5vm01,
│   on .terraform\modules\bigip\main.tf line 260, in resource "google_compute_instance" "f5vm01":
│  260:   provisioner "local-exec" {
│
│ Error running command 'sleep 250': exit status 1. Output: 'sleep' is not recognized as an internal or external
│ command,
│ operable program or batch file.
│
╵
PS C:\Users\memes\Documents\1nic>
```

**Comment by memes** [comment]
PR #11 offers a fix for the issue.

**Comment by trinaths** [comment]
Created INFRAANO-607 for internal tracking.

**Comment by RavinderReddyF5** [comment]
@memes fixed in v1.1.0
